src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Definition/DefinitionEndpoint.cs

```diff
@@ -51,6 +51,8 @@ internal sealed class DefinitionEndpoint : AbstractRazorDelegatingEndpoint<TextD
 
     protected override string CustomMessageTarget => RazorLanguageServerCustomMessageTargets.RazorDefinitionEndpointName;
 
+    protected override string LspTarget => Methods.TextDocumentDefinitionName;
```

Comment:
We should have a constant defined in each handler, that this property and the attribute use, so there is no chance of them getting out of sync.

So something like (I've taken the liberty of changing `LspTarget` to just `MethodName` - having "Lsp" in there seems a little redundant)
```cs
[LanguageServerEndpoint(LspEndpointName )]
internal sealed class DefinitionEndPoint
{
    public const string LspEndpointName = Methods.TextDocumentDefinitionName;

   //..

    protected override string MethodName => LspEndpointName;
```

Alternatively we could probably just get the method name from reflection, as long as it happens at startup and not on each request, it wouldn't be the worst thing ever.